[2024-04-19T15:17:12.421+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T15:17:07.368351+00:00 [queued]>
[2024-04-19T15:17:12.429+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T15:17:07.368351+00:00 [queued]>
[2024-04-19T15:17:12.430+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 3
[2024-04-19T15:17:12.440+0000] {taskinstance.py:2217} INFO - Executing <Task(PythonOperator): run_power_data_manipulation> on 2024-04-19 15:17:07.368351+00:00
[2024-04-19T15:17:12.444+0000] {standard_task_runner.py:60} INFO - Started process 76 to run task
[2024-04-19T15:17:12.447+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'my_data_pipeline', 'run_power_data_manipulation', 'manual__2024-04-19T15:17:07.368351+00:00', '--job-id', '335', '--raw', '--subdir', 'DAGS_FOLDER/my_data_pipeline.py', '--cfg-path', '/tmp/tmp392g2hr5']
[2024-04-19T15:17:12.449+0000] {standard_task_runner.py:88} INFO - Job 335: Subtask run_power_data_manipulation
[2024-04-19T15:17:12.500+0000] {task_command.py:423} INFO - Running <TaskInstance: my_data_pipeline.run_power_data_manipulation manual__2024-04-19T15:17:07.368351+00:00 [running]> on host 3e7a91784722
[2024-04-19T15:17:12.578+0000] {taskinstance.py:2513} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_data_pipeline' AIRFLOW_CTX_TASK_ID='run_power_data_manipulation' AIRFLOW_CTX_EXECUTION_DATE='2024-04-19T15:17:07.368351+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-19T15:17:07.368351+00:00'
[2024-04-19T15:17:12.580+0000] {logging_mixin.py:188} INFO - No TXT file found in the directory
[2024-04-19T15:17:12.580+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-04-19T15:17:12.589+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=my_data_pipeline, task_id=run_power_data_manipulation, execution_date=20240419T151707, start_date=20240419T151712, end_date=20240419T151712
[2024-04-19T15:17:12.620+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-19T15:17:12.638+0000] {taskinstance.py:3312} INFO - 0 downstream tasks scheduled from follow-on schedule check
